docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -110,7 +110,7 @@ This value is usually set to `6` for Gregorian calendars, because months display
 ### Add week number
 
 To display week number, use the `displayWeekNumber`.
-If you use a `dateAdapter` imported from outside of our library, you will have to provide the method `getWeekNumber`.
+If you do not import your `dateAdapter` from `@mui/x-date-pickers` or `@mui/x-date-pickers-pro`, you will have to provide the method `getWeekNumber`.
```

Comment:
Now that we know we will force usage of adapters imported from our packages, is this `getWeekNumber` method relevent ?